windows.md

```diff
@@ -390,6 +390,49 @@ With those compatibility things out of the way, you're ready to start the system
     4. Open the start menu and search for "Ubuntu". Start it - it should ask you to create a user with a password. This will be your user to log in to your Ubuntu Linux Subsystem - note down the username and password somewhere secure to make sure you do not forget it.
     5. Open the start menu and search for "Docker Desktop". Start it and go to the Settings. Under the General tab, you will find an option called "Use WSL 2 based engine". Make sure this is checked.
 
+    If opening Docker Desktop from **Option A or B** shows an error, you may need to enable virtualization on your machine
+
+    The error might look like this:<br><br>
+    <img src="docker-desktop-opening-error.avif"><br><br>
+
+    Close PowerShell and open it again as administrator (like in step 2)
+
+    Check if your computer supports virtualization. Run this command:
+
+    ```bash
+    Get-ComputerInfo -Property HyperV*
+    ```
+
+    This should display the following output:
+
+    ```bash
+    HyperVisorPresent: False
+    HyperVRequirementDataExecutionPreventionAvailable : True
+    HyperVRequirementSecondLevelAddressTranslation    : True
+    HyperVRequirementVirtualizationFirmwareEnabled    : False
+    HyperVRequirementVMMonitorModeExtensions          : True
+    ```
+
+    Check if virtualization is enabled. Run this command:
+
+    ```bash
+    systeminfo | findstr Virtualization
+    ```
+
+    This should display the following output:
+
+    ```bash
+    Virtualization Enabled In Firmware: No
+    ```
+
+    Continue with the steps below if virtualization is not enabled to enable virtualization in your BIOS/UEFI:
+
+    1. Restart your computer
+    2. Before the brand logo appears, press the BIOS/UEFI access key repeatedly. Common options are <kbd>Delete</kbd>, <kbd>Esc</kbd>, <kbd>F1</kbd>, <kbd>F2</kbd>, <kbd>F4</kbd>, <kbd>F10</kbd>, or <kbd>F12</kbd> or as shown on the startup screen or check this [BIOS/UEFI key lists](https://www.disk-image.com/faq-bootmenu.htm)
+    3. Navigate to the `BIOS/UEFI Setup Utility`
+    4. Find the virtualization option for your computer and enable it
```

Comment:
let's:

- be a bit more clear that the student will need to search for it
- check if there are any "more common" names that we can add here, just like we added "more common" hotkeys above
- offer a link to some good documentation about what the names of this would be

Reply:
I can't find any documentation that centralizes all the virtualization options. I have the link from Microsoft, but they will still need to click around on the lists of the computer manufacturers there.
https://support.microsoft.com/en-us/windows/enable-virtualization-on-windows-11-pcs-c5578302-6e43-4b4b-a449-8ced115f58e1

Some other options were
https://techlibrary.hpe.com/docs/iss/proliant_uefi/UEFI_TM_030617/s_Virtualization_options.html

f467126fa9850475d169fe0cdc53aa0d9ca4c7e7